### ISSUE TITLE
ask some question

### ISSUE THREAD

**Issue by TyrionChou** [nadarasarbahavan/SpHOR#1]
Thank you very much for your work. I have a few questions to ask:

1.For very similar characters, such as "大" and "太" in Chinese, can the features in the first step distinguish between them?

2.If the training set contains "大" but the open-set character is "太," is there a possibility that "太" could be classified as "大" with a high prediction score and fail to be rejected?

3.For instances of "大" in the test set that are not very similar to those in the training set, could they produce a relatively low prediction score, leading to incorrect rejection during testing?

**Comment by nadarasarbahavan** [comment]
1. This work focuses on discriminative learning, so it learns the most discriminative/unique features between the classes within the training class set. As result if the training set has both those characters then it can learn to distinguish between them, very well. 
2. the goal of the model is to learn the most unique features.  the type of unique feature it learns depends on the other classes in the training dataset. if there is enough variety in the training dataset, it can learn very strong unique features: thus, work well in rejecting your openset character. 
3. Yes. the softmax probability/logit score will be slightly lower for those types of images due to their difficulty. However it will not be low such that it gets rejected. If so, you can adjust the lamda parameter to be lower so the diffucult cases pass through. Also it depends on the relative cost: is rejecting the difficult "大" more costly than passing the unknown as known?
